testsuite/tests/statmemprof/lists_in_minor.ml

```diff
@@ -1,6 +1,10 @@
 (* TEST
    flags = "-g"
    * bytecode
+     reference = "${test_source_directory}/lists_in_minor.byte.reference"
+   * native
+     reference = "${test_source_directory}/lists_in_minor.opt.reference"
+     compare_programs = "false"
```

Comment:
The `minor_no_postpone` test needs to be activated in native mode too.

Reply:
Thanks, missed this.